### PR TITLE
Added income from investments table rows and details tasks

### LINKED ISSUE #84: [MICROTASK] Introducere informatii rand tabel VII. Venituri ale declarantului, etc - sectiunea 4. Venituri din investitii
Microtask pentru introducerea informatiilor din tabelul VII. Venituri ale declarantului, etc - Sectiunea 4.Venituri din investitii. Acest microtask trebuie generat pentru fiecare rand din tabelul de mai sus. Acest microtask trebuie generat in metoda de after_save in cadrul taskului definit in #27 

Microtaskul va primi ca si parametru numarul randului din tabelul mai sus mentionat. Implementarea microtaskului va contine:

Presenter:

Afiseaza pdf ul, se face scroll sau highlight pe tabelul VII. Venituri ale declarantului si ale membrilor sai de familie - sectiunea 4. Venituri din investitii folosind fie OCR fie prin mesaj direct catre vountar, implementat in Form

Form:

Afiseaza mesaj catre voluntar in cazul in care highlightul nu se face folosind solutie de OCR

"Va rugam introduceti informtiile de pe randul {nr_rand} completat din tabelul VII. Venituri ale declarantului si ale membrilor sai de familie - sectiunea 4. Venituri din investitii"

Afiseaza urmatoarele elemente care trebuie completate de catre voluntar din informatiile din tabel
Dropdown pt selectie Relatie titular - ENUM 
Textbox pt introducere Nume Persoana (Persoana)
Textbox pt introducere Prenume Persoana (Persoana)
Textbox pt selectie Sursa venit
Dropdown pt selectie Judet pt adresa - ENUM 
Dropdown pt selectie Localitate pt adresa - ENUM 
Textbox pt introducere Comuna pt adresa
Textbox pt introducere Adresa strainatate
Textbox pt introducere Serviciu prestat
Dropdown pt introducere Functie - ENUM
Textbox pt introducere Venit anual incasat
Dropdown pt selectie Valuta - ENUM

Cross-Verification

Pt fiecare din campuri vom avea criteriul de verificare si acceptare ca fiind egalitatea valorilor introduse de voluntari

Save Verified DataSe va salva acest db object care tb definit in models.py

Venituri

Relatie_titular (ENUM)
Nume (STRING)
Prenume (STRING)
Adresa (STRING)
Sursa (ENUM)
Institutie (STRING)
Serviciu_prestat (STRING)
Suma (NUM)
Valuta (ENUM)

Persoana
Nume (STRING)
Prenume (STRING)

### PR DESCRIPTION
**Pull request by danielacraciun**
Fixes #84

### CODE REVIEW

File: project_template/forms.py [hunk 1]
```diff
@@ -173,6 +173,20 @@ class TranscribeOwnedIncomeFromInvestmentsTable(forms.Form):
...
+class TranscribeOwnedIncomeFromInvestmentsRowEntry(forms.Form):
+    holder_relationship = forms.ChoiceField(label="Cine este beneficiarul venitului din investitii?", choices=HolderRelationship.return_as_iterable())
+    surname = forms.CharField(label="Care e numele persoanei?")
+    name = forms.CharField(label="Care e prenumele persoanei?")
+    source = forms.CharField(label="Care este sursa venitului?")
+    county = forms.ChoiceField(label="Care este judetul de unde provine sursa de venit?", choices=Counties.return_counties())
+    city = forms.CharField(label="Care este localitatea de unde provine sursa de venit?")
+    commune = forms.CharField(label="Care este comuna de unde provine sursa de venit?")
+    offered_service = forms.CharField(label="Care e serviciul prestat?")
+    position = forms.ChoiceField(label="Care a fost functia detinuta?", choices=Position.return_as_iterable())
```
Comment (catileptic) [inline project_template/forms.py]: This is redundant. The `offered_service` variable already holds the information. Please remove it.

The header of the table, for this column, is `Serviciul prestat / Obiectul generator de venit`. Incidentally, the "service" is a position that this person holds within an institution. But this is, more abstractly, a "service" (so already covered by the previous variable).

File: project_template/models.py [hunk 2]
```diff
@@ -292,6 +292,8 @@ class OwnedIncomeFromInvestmentsTable(models.Model):
 # Tabel Venituri investitii - actual row information
 class OwnedIncomeFromInvestmentsTableEntry(CommonIncomeFields):
     table = models.ForeignKey(OwnedIncomeFromInvestmentsTable, on_delete=models.CASCADE, null=True)
+    receiver_of_goods = models.ForeignKey(Person, on_delete=models.CASCADE, null=True)
+    position = models.CharField("Pozitia detinuta", max_length=128)
```
Comment (catileptic) [inline project_template/models.py]: See comment regarding the `position` variable in `forms.py`. This is redundant. Please, remove it.

File: project_template/models.py [hunk 3]
```diff
@@ -292,6 +292,8 @@ class OwnedIncomeFromInvestmentsTable(models.Model):
 # Tabel Venituri investitii - actual row information
 class OwnedIncomeFromInvestmentsTableEntry(CommonIncomeFields):
     table = models.ForeignKey(OwnedIncomeFromInvestmentsTable, on_delete=models.CASCADE, null=True)
+    receiver_of_goods = models.ForeignKey(Person, on_delete=models.CASCADE, null=True)
```
Comment (catileptic) [inline project_template/models.py]: The mapping between the political figure who is the owner and the beneficiary of all the objects inside the asset declaration and the asset declaration itself is already implemented. 

The implementation works as follows: a `Declaration` model has a `Politician` as a foreign key ("One politician can have one or more declarations"), a `*Table` model (for example `OwnedInvestmentsTable`) has a `declaration` model as a foreign key ("One declaration can have multiple tables, among which one is a `OwnedInvestmentsTable` table).

Comment (danielacraciun) [reply project_template/models.py]: So it also doesn't make any sense to have name and surname in the corresponding form, right?

Comment (danielacraciun) [reply project_template/models.py]: So the name and surname retrieved from this form should go through this mapping, and not have a new one here.

**Pull request merged**